pkcs11/util_pkcs11.c

```diff
@@ -3151,7 +3149,7 @@ CK_RV parse_ec_template(CK_ATTRIBUTE_PTR pTemplate, CK_ULONG ulCount,
                         yubihsm_pkcs11_object_template *template) {
 
   uint8_t *ecparams = NULL;
-  uint16_t ecparams_len;
+  uint16_t ecparams_len = 0;
```

Comment:
is this change needed?

Reply:
MSVC warning that the variable is not initialized